src/Sarif/FileRegionsCache.cs

```diff
@@ -142,6 +142,9 @@ private Region PopulateTextRegionProperties(NewLineIndex lineIndex, Region input
             return region;
         }
 
+        internal const int BIGSNIPPETLENGTH = 512;
```

Comment:
>BIGSNIPPETLENGTH [](http://example.com/codeflow?start=27&length=16)

Note that I've lifted out these magic values to make them more explicit in test use, and to allow product changes to flow more naturally to test validation.

Reply:
do we prefer put at the top of the file with other definitions

Reply:
Yes, will do this in a follow-on change, thanks for the review!!